README.md

```diff
@@ -1 +1,22 @@
-# UIKTP
\ No newline at end of file
+# UIKTP
+
+## Project setup
+To setup the project, first create a virtual environment using the following commands:
+    
+    python -m venv furryfeet_env
+
+Then, activate the created env with the command
+
+    furryfeet_env/bin/activate  
```

Comment:
Maybe we should add furryfeet_env/Scripts/activate as most of us are windows users

Reply:
minus points for my copy/paste abilities. i'll update it